cypress/e2e/awx/administration/instanceGroups.cy.ts

```diff
@@ -25,6 +27,133 @@ describe('Instance Groups', () => {
       //Assert deletion of the Instance Group
       //Delete the IG created in this test
     });
+    it('can create a Container Group, assert information on the details page, and then delete the instance group', () => {
+      const containerGroupName = 'E2E Instance Group' + randomString(4);
+      const credentialName = 'E2E Credential ' + randomString(4);
+      let organization: Organization;
+
+      let credential: Credential;
+      cy.createAwxOrganization().then((org) => {
+        organization = org;
+
+        cy.createAWXCredential({
+          kind: 'kubernetes_bearer_token',
+          organization: organization.id,
+          credential_type: 17,
+          name: credentialName,
+        }).then((cred) => {
+          credential = cred;
+          cy.navigateTo('awx', 'instance-groups');
+          cy.clickButton(/^Create group$/);
+          cy.clickLink(/^Create container group$/);
+          cy.get('#credential-select').type(credential.name);
+          cy.get('[data-cy="name"]').type(containerGroupName);
+          cy.getByDataCy('override').click({ force: true });
+          cy.get('[data-cy="max-concurrent-jobs"]').clear();
+          cy.get('[data-cy="max-concurrent-jobs"]').type('3');
+
+          cy.get('[data-cy="max-forks"]').clear();
+          cy.get('[data-cy="max-forks"]').type('4');
+          cy.clickButton(/^Create Container Group$/);
+          cy.verifyPageTitle(containerGroupName);
```

Comment:
We should delete the resources created by the test.

Reply:
As far as deleting, there is a delete function in the afterEach block.